app/controllers/admin/api/application_plan_limits_controller.rb

```diff
@@ -14,15 +14,19 @@ class Admin::Api::ApplicationPlanLimitsController < Admin::Api::BaseController
   ##~ op.description = "Returns the list of all limits associated to an application plan."
   ##~ op.group = "application_plan_limits"
   #
+  ##~ @parameter_per_page_50 = {:name => "per_page", :description => "Number of results per page. Default and max is 50.", :dataType => "int", :paramType => "query", :defaultValue => "50"}
+  #
   ##~ op.parameters.add @parameter_access_token
+  ##~ op.parameters.add @parameter_page
+  ##~ op.parameters.add @parameter_per_page_50
   ##~ op.parameters.add @parameter_application_plan_id_by_id_name
   #
-
   def index
-    respond_with(usage_limits)
+    respond_with(usage_limits.paginate(page: current_page, per_page: per_page))
```

Comment:
Just for consistency, I'd use the same names
```suggestion
    respond_with(usage_limits.paginate(current_page: current_page, per_page: per_page))
```